connectors/sample/README.md

```diff
@@ -0,0 +1,127 @@
+# Asset(Sample) ISC-CAR
+
+Asset(Sample) CAR Connector
+```
+I.   SUMMARY
+II.  PREREQUISITES
+III. INSTALLATION
+IV.  EXAMPLE USAGE (command line)
+V.   INITIAL IMPORT
+```
+
+I. SUMMARY:
+-----------------------------------------------------------------
+This connector simplifies the process of loading data into CAR. Users can import data from files directly into CAR without the need to manage servers, resulting in faster data loading times.
```

Comment:
I know this is a little nit-picky but this does require running the reference connector as a server for data upload.

Reply:
Not necessarily. While we can retrieve the data by running the server, the connector itself only loads data from files